File: src/main/old/bbarena/model/event/game/ThrowInBallEvent.java
package bbarena.model.event.game;

/**
 * Created by Teg on 02/11/16.
 */
public class ThrowInBallEvent extends ScatterBallEvent {

    public ThrowInBallEvent(int ballId) {
        super(ballId);
    }

    public ThrowInBallEvent(int ballId, int direction) {
        super(ballId, direction);
    }

    public ThrowInBallEvent(int ballId, int direction, int distance) {
        super(ballId, direction, distance);
    }
}

File: src/main/kotlin/bbarena/model/event/game/ThrowInBallEvent.kt
package bbarena.model.event.game

/**
 * Created by Teg on 02/11/16.
 */
class ThrowInBallEvent : ScatterBallEvent {

    constructor(ballId: Int) : super(ballId) {}

    constructor(ballId: Int, direction: Int) : super(ballId, direction) {}

    constructor(ballId: Int, direction: Int, distance: Int) : super(ballId, direction, distance) {}
}
